device/bgp/bgp.yaml

```diff
@@ -114,3 +114,158 @@ components:
+              x-field-uid: 3
+
+    Device.Bgp.OpenMessageError:
+      description: >-
+        All errors detected while processing the processing the OPEN message are indicated by sending the NOTIFICATION message 
+        with the Error Code-Open Message Error. The Error Subcode elaborates on the specific nature of the error.
+      type: object
+      properties:
+        subcode:
+          description: >-
+            The Error Subcode, the specific type of error encountered during OPEN message processing.
+          type: string
+          x-field-uid: 1
+          x-enum:
+            unsupported_version_number_code2_subcode1:
+              x-field-uid: 1
+            error_peer_as_code2_subcode2:
+              x-field-uid: 2
+            error_bgp_id_code2_subcode3:
+              x-field-uid: 3
+            unsupported_optional_parameter_code2_subcode4:
+              x-field-uid: 4
+            auth_failed_code2_subcode5:
+              x-field-uid: 5
+            unsupported_hold_time_code2_subcode6:
+              x-field-uid: 6
+            unsupported_capability_code2_subcode7:
+              x-field-uid: 7
+
+    Device.Bgp.UpdateMessageError:
+      description: >-
+        All errors detected while processing the processing the UPDATE message are indicated by sending the NOTIFICATION message 
+        with the Error Code-Update Message Error. The Error Subcode elaborates on the specific nature of the error.
+      type: object
+      properties:
+        subcode:
+          description: >-
+            The Error Subcode, the specific type of error encountered during UPDATE processing.
+          type: string
+          x-field-uid: 1
+          x-enum:
+            malformed_attrib_list_code3_subcode1:
+              x-field-uid: 1
+            unrecognized_wellknown_attrib_code3_subcode2:
+              x-field-uid: 2
+            wellknown_attrib_missing_code3_subcode3:
+              x-field-uid: 3
+            attrib_flags_error_code3_subcode4:
+              x-field-uid: 4
+            attrib_length_error_code3_subcode5:
+              x-field-uid: 5
+            invalid_origin_attrib_code3_subcode6:
+              x-field-uid: 6
+            as_routing_loop_code3_subcode7:
+              x-field-uid: 7
+            invalid_nhop_attrib_code3_subcode8:
+              x-field-uid: 8
+            error_optional_attrib_code3_subcode9:
+              x-field-uid: 9
+            invalid_network_field_code3_subcode10:
+              x-field-uid: 10
+            abnormal_aspath_code3_subcode11:
+              x-field-uid: 11
+          
+    Device.Bgp.HoldTimerExpired:
+      description: >-
+        If a system does not receive successive KEEPALIVE, UPDATE, and/or NOTIFICATION messages within the period specified 
+        in the Hold Time field of the OPEN message, then the NOTIFICATION message with the Hold Timer Expired Error Code is 
+        sent and the BGP connection is closed.The sub code used by default is 0.
```

Comment:
Maybe just specify that the Hold Timer Expired notification will always be sent with Error Code x and Error Subcode 0. [ If we remove subcode ] 
or 
At least mention Error Value that will be used. ( since even if subcode is included, it is not an enum so Error Code value is not available /known only via looking model/help for this object )

Reply:
removed subcode property from hold timer error and finite state machine error